google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/sql_objects/sql_objects_metadata_config.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.datacatalog_connectors.commons.config import yaml_config
+
+from schema import Schema, And, Optional
```

Comment:
```suggestion
import schema
```

Reply:
Doing this I think the code becomes more verbose, I think it is easier to read this way:

  ```
      {
            __METADATA_DEFINITION_KEY: {
                'name': And(str),
                'purpose': And(str),
                Optional('inputs'): [{
                    'name': And(str),
                    'type': And(str)
                }],
                Optional('outputs'): [{
                    'name': And(str),
                    'type': And(str)
                }]
            }
        }
```

Otherwise we would have `schema` being repeeated multiple times. WDYT?